src/PushCommand.php

```diff
@@ -389,8 +392,13 @@ private function getIgnores(InputInterface $input)
         $optionalIgnore = $input->getOption('ignore');
         $composerIgnores = $this->getNexusExtra('ignore', []);
         $gitAttrIgnores = $this->getGitAttributesExportIgnores($input);
-        $composerJsonIgnores = $this->getComposerJsonArchiveExcludeIgnores($input);
-        $defaultIgnores = ['vendor/'];
+	$composerJsonIgnores = $this->getComposerJsonArchiveExcludeIgnores($input);
```

Comment:
The number of spaces are not the same, php lint ;) You used tab instead of spaces